content/post/2017-09-17-digest-nushackers-site-redesign.md

```diff
@@ -0,0 +1,44 @@
+
+## Migrating
+
+When the proposal was raised to redesign the site, that idea seemed incredibly difficult because Jekyll, which was written in Ruby, took roughly 11 seconds for a single change to reflect on the browser, which was simply too long and frustrating. As a result, we decided to move to [Hugo](https://gohugo.io).
+
+{{< imglink src="/img/2017/09/hugo.png" alt="gohugo.io" >}}
+<br/>
+
+Hugo boasted about its speed and it proved itself when it would take less than a millisecond to rebuild our 500 page blog. it was an astounding change and made developing much easier and convenient.
+
+Hugo also provided a command line tool to migrate from Jekyll to Hugo, which made the migration incredibly easy - `hugo import jekyll` was all it took. However that was not to say that there weren't any difficulties encountered along the way, an easily overlooked feature in Jekyll was that templates could be placed anywhere, even within markdown itself. That was not available in Hugo, and we had to make use of a feature called shortcodes to replace the templates that we had. Shortcodes were simple tags you could write to provide functionality, but they were not as flexible as inlined templates
+
+## Maintainable css
+
+Another feature that was lost was support for [scss](http://sass-lang.com/), a superset of css, which makes css easier to maintain.
+
+<img src="/img/2017/09/scss.png" alt="http://sass-lang.com/"/>
+
+We managed to add it back with a build process which also concurrently spawns the Hugo instance. The build process, written in a javascript library called Gulp, allowed us to work on styling the site where every change would automatically reload the browser to reflect the change. In this build process, we also included libraries Prettier and Stylelint-order, which auto-formatted scss. This meant we would never have to worry about how our scss code was written, creating stress-free code reviews.
+
+## Faster site & build previews
+
+The last feature that we added was to move our hosting to a static site called [Netlify](https://netlify.com).
+
+<img src="/img/2017/09/netlify.png" alt="netlify"/>
+
+Not only did it improve the load time of our site through their incredibly fast Content-Distribution Networks (CDN), they also provided build previews whenever we pushed to to github. This meant that any new changes could be previewed in an actual fork of the site. It certainly made reviewing a lot easier and more trustworthy.
+
+When all said and done, the new blog is incredibly, fast, convenient and beautiful. We hope that this digest encouraged you to start your own. Our code is always publically available at [Github](github.com/nushackers/nushackers-site) and you are free to copy if for your own use!
```

Comment:
- Remove 'When'
- 'Publicly'
- 'it for your own use'

Reply:
Also "incredibly, fast" --> "incredibly fast"